### DISCUSSION TITLE
Broken link in warning message regarding hinting the OS/driver to use discrete graphics adapter

### DISCUSSION THREAD

**Discussion by Alan-FGR** [DiligentGraphics/DiligentCore#461] [General]
Hello guys! I'm not creating an issue for this since it seemed overkill and there's probably some discussion involved.

The error message [here](https://github.com/DiligentGraphics/DiligentCore/blob/020431c6ffd5cb93e8d6a561fc8ac3536a226305/Graphics/GraphicsEngineOpenGL/src/EngineFactoryOpenGL.cpp#L151) has a broken link. Basically it was a gist that would show how to expose the symbols that hint the OS/driver to use the discrete GPU (for systems with both integrated and dedicated).

Basically the snippet was something like this:
```
extern "C"
{
    __declspec(dllexport) uint32_t NvOptimusEnablement = UINT32_C(1);
    __declspec(dllexport) uint32_t AmdPowerXpressRequestHighPerformance = UINT32_C(1);
}
```
Which need to be exported from the executable binary.

I'm not sure what other link could replace this one, but maybe it could be in 1st party docs so there's no risk of stuff breaking again?

**Comment by MikhailGorobets** [comment]
OpenGL doesn't allow to select a discrete graphics adapter by default, so you need to export these variables in the executable or you can use this way
https://github.com/DiligentGraphics/DiligentSamples/blob/aba2598b743489f0ee31311bd6e9c3cb2e942e70/Tutorials/Tutorial27_PostProcessing/CMakeLists.txt#L40
Next you need to set the values of these variables to 1, you can do it like this
https://github.com/DiligentGraphics/DiligentSamples/blob/aba2598b743489f0ee31311bd6e9c3cb2e942e70/Tutorials/Tutorial27_PostProcessing/src/Tutorial27_PostProcessing.cpp#L649
The link is correct, it's just that an error appeared there during code refactoring. We need to remove `XpressRequestHighPerformance` from the link

**Comment by TheMostDiligent** [comment]
The link (https://gist.github.com/statico/6809850727c708f08458XpressRequestHighPerformance) is indeed broken. @MikhailGorobets can you fix it?

**Reply by MikhailGorobets** [reply]
https://github.com/DiligentGraphics/DiligentCore/pull/462

**Comment by TheMostDiligent** [comment]
Fixed in https://github.com/DiligentGraphics/DiligentCore/pull/462
